app/Http/Middleware/VerifyQuiz.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+
+class VerifyQuiz
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @return mixed
+     */
+    public function handle($request, Closure $next)
+    {   
+        if ($request->session()->has('quiz_token')) {
```

Comment:
you can use session directly as a facade no need to chain it with request object.